app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/AlertPlayer.java

```diff
@@ -60,8 +59,10 @@ public synchronized  void startAlert(Context ctx, boolean trendingToAlertEnd, Al
 
         stopAlert(ctx, true, false);
         int alertIn = newAlert.minutes_between;
-        if(alertIn < 1) { alertIn = 1; }
-        ActiveBgAlert.Create(newAlert.uuid, false, new Date().getTime() + alertIn * 60000 );
+        if(alertIn < 1 || AlertPlayer.isAscendingMode(ctx)) { alertIn = 1; }
+        ActiveBgAlert aba = ActiveBgAlert.Create(newAlert.uuid, false, new Date().getTime() + alertIn * 60000);
+        aba.last_alerted_at = System.currentTimeMillis();
+        aba.save();
```

Comment:
Create() is also called in other occasions like PreeSnooze e.g.
Not knowing the logic behind those, I just wanted to set the time the alarm last rang just in those cases, it actually rang. Feel free to change that if you are sure it doesn't cause side effects.